rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -105,6 +106,7 @@
   private static final int DEFAULT_SERVER_MAX_THREADS = Integer.MAX_VALUE;
   private static final int DEFAULT_SERVER_PORT = 8899;
   private static final int DEFAULT_SOCKET_READ_TIMEOUT = 30000; // ms
+  private static final int DEFAULT_CACHE_CONCURRENCY_LEVEL = 4;
```

Comment:
A way to turn off the change is required. Can the default value be 1 ? That way the change is not _rolled out_. Based on an account feature we can set it to 4. For any other value, a bootstrap can be used.

Reply:
The value has been 4. We are not changing the default value. We just changed the weight for the dummy key.
The default value can be found here.
https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/CacheBuilder.java#L159

Reply:
Hmm. I am out of my depth here. I interpreted the change as:
- Before _getWeight_ function returned 100 but now returns 25
- There maybe unintended consequences due to the change in weight for the same _usedMb_. 
- If that is true, then a roll out is requored.